### PR TITLE
Fix linter warning message

### PR DESCRIPTION
**Pull request by s1061123**
Fix linter/revive warning message.

### CODE REVIEW

**Review by maiqueb — APPROVED** [review]
Thank you!

File: pkg/server/thick_cni_test.go [hunk 1]
```diff
@@ -221,10 +221,7 @@ func prepareCNIEnv(netnsPath string, namespaceName string, podName string, podUI
 	if err := os.Setenv("CNI_NETNS", netnsPath); err != nil {
 		return err
 	}
-	if err := os.Setenv("CNI_ARGS", cniArgs); err != nil {
-		return err
-	}
-	return nil
+	return os.Setenv("CNI_ARGS", cniArgs)
```
Comment (nicklesimba) [inline pkg/server/thick_cni_test.go:224]: Is CNI_ARGS going to be the new default instead of returning nil? And what is the reason for this change?

File: pkg/server/thick_cni_test.go [hunk 2]
```diff
@@ -46,17 +46,17 @@ const suiteName = "Thick CNI architecture"
...
 
 // ExecPlugin executes the plugin
-func (fe *fakeExec) ExecPlugin(ctx context.Context, pluginPath string, stdinData []byte, environ []string) ([]byte, error) {
+func (fe *fakeExec) ExecPlugin(_ context.Context, _ string, _ []byte, _ []string) ([]byte, error) {
 	return []byte("{}"), nil
 }
 
 // FindInPath finds in path
-func (fe *fakeExec) FindInPath(plugin string, paths []string) (string, error) {
+func (fe *fakeExec) FindInPath(_ string, _ []string) (string, error) {
```
[the remark this answers is not in the document]
Comment (maiqueb) [reply pkg/server/thick_cni_test.go:54]: Make peace with the linter ? It is complaining these attributes are not being used.

Thus we tell the compiler we don't care enough to name them. 

... this needs to happen because this is a method of an interface.

[the remark this answers is not in the document]
Comment (nicklesimba) [reply pkg/server/thick_cni_test.go:54]: Thanks Miguel, also talked to Tomo about the changes. I think we're good to merge here

**Review by nicklesimba — APPROVED** [review]
Changes looks good to me, and thanks Tomo/Miguel for explaining the need for these changes.
